### ISSUE TITLE
TypeError: mb_strtolower(): Argument #1 ($string) must be of type string, array given in src/Css/Style.php:3003

### ISSUE THREAD

**Issue by cedx** [dompdf/dompdf#2338]
With PHP 7.4 and DOMPDF 0.8.6, everything is fine. After upgraded to PHP 8.0 and DOMPDF 1.0.0, I get the following error:

```
Fatal error:  Uncaught TypeError: mb_strtolower(): Argument #1 ($string) must be of type string, array given in vendor/dompdf/dompdf/src/Css/Style.php:3003
Stack trace:
#0 vendor/dompdf/dompdf/src/Css/Style.php(3003): mb_strtolower(Array)
#1 vendor/dompdf/dompdf/src/Css/Style.php(893): Dompdf/Css/Style->set_size(Array)
#2 vendor/dompdf/dompdf/src/Css/Style.php(798): Dompdf/Css/Style->__set('size', Array)
#3 vendor/dompdf/dompdf/src/Css/Stylesheet.php(1401): Dompdf/Css/Style->merge(Object(Dompdf/Css/Style))
#4 vendor/dompdf/dompdf/src/Css/Stylesheet.php(332): Dompdf/Css/Stylesheet->_parse_css('@page { margin:...')
#5 vendor/dompdf/dompdf/src/Dompdf.php(685): Dompdf/Css/Stylesheet->load_css('/r/n/t/t/t@page { ma...', 3)
#6 vendor/dompdf/dompdf/src/Dompdf.php(764): Dompdf/Dompdf->processHtml()
#7 lib/mc2it_labels/server/labeling/LabelController.php(146): Dompdf/Dompdf->render()
```

The HTML page rendered has the following `<style>` content:

```
@page {
	margin: 0;
	padding: 0;
	size: 10cm 14.9cm portrait;
}
```

**Comment by bsweeney** [comment]
Dompdf 0.8.6 does not support PHP 8 (see #2242). You'll need to upgrade to 1.0.0 (or if you can wait 1.0.1 because of some bugs in 1.0.0).

**Comment by cedx** [comment]
Sorry to bother you: I'm using v1.0.0 (read carefully the issue).
I'll wait for v1.0.1 to see if this issue is really fixed. In the meantime: thanks for your efforts.

**Comment by bsweeney** [comment]
Ah ha ha sorry 😬 ... lots going on right now so I did skim instead of read carefully.

That is indeed a bug in the 1.0.0 release and is related to #2334 (addressed by commit 80424ef). The 1.0.1 release will include a fix.

**Comment by cedx** [comment]
@bsweeney I've just tested v1.0.1 and now everything works as expected. Congratulations, and thanks a lot!
